k8s/Dockerfile

```diff
@@ -27,7 +27,7 @@ ARG RAPIDS_VER=22.12.0
 ARG hadoop_aws_ver=3.3.1
 ARG aws_java_sdk_bundle_ver=1.11.655
 ARG gcs_connector_ver=2.2.2
-
+ARG delta_core_ver=2.0.2
```

Comment:
According to https://github.com/NVIDIA/spark-rapids/blob/branch-23.02/delta-lake/README.md, 2.0.x delta lake works with Spark 3.2.x, and 2.1.x+ works with Spark 3.3.x.
We'd better to add some text in README to mention the compatibility of Delta Lake and Spark.
And BTW, we'd like to use the latest Spark version 3.3.1 as the default. Can we update the version accordingly?

Reply:
Done, and verified on EKS.